scriptshifter/tables/data/index.yml

```diff
@@ -64,6 +53,17 @@ church_slavonic:
 chuvash_cyrillic:
   marc_code: chv
   name: Chuvash (Cyrillic)
+cyrillic_generic:
+  description: 'Multi-purpose transliteration for most languages that use the Cyrillic script:
```

Comment:
I see languages that already have their own specific table in this list. Should they be removed? This message is only informational but it can be misleading.